vnext/overrides.json

```diff
@@ -252,10 +252,10 @@
       "baseHash": "e982deb23d7ad5312ab0c09508a6d58c152b5be7"
     },
     {
-      "type": "derived",
+      "type": "copy",
```

Comment:
Core has a builtin stub we were also using in react-native-win32, but we had replaced it with UnimplementedView for some reason. Changed to just use the builtin stub.